Misc/NEWS.d/next/Library/2018-08-16-16-47-15.bpo-20849.YWJECC.rst

```diff
@@ -0,0 +1,2 @@
+shutil.copytree now accepts a new ``dirs_exist_ok`` keyword argument.
+Patch by Josh Bronson.
```

Comment:
Please also update `Doc/whatsnew/3.8.rst` by adding a section re. the improvements to the `shutil` module.

Reply:
Good call, done in the latest revision.